### ISSUE TITLE
Missing spaces in html attributes

### ISSUE THREAD

**Issue by FrederikLive** [PrestaShop/PrestaShop#20059]
Several scripts dynamicly adds attributes to html in Admin where spaces are not included.

One example is: /Admin/ProductPage/Blocks/header.html.twig
Opened by Admin->sell/catalog/products->Add New Product

Around:
```
language.iso_code %} selected="selected"{% endif %}>{{language.iso_
```
-> %}>**(NO_SPACE_HERE)**{{

I've found multiple of these errors which breaks layout and usability in browser. Tested in PrestaShop 1.7.6.5 and 1.7.6.6. Both new Installations.

More found under Add New Product (Just a few):
```
<input type="text"id="form_step1_name_1" name="form[step1][name][1]" required="required" placeholder="Enter your product name" class="edit js-edit serp-default-title form-control"/>

<input type="text"id="form_step1_name_2" name="form[step1][name][2]" placeholder="Enter your product name" class="edit js-edit serp-default-title form-control"/>

option value="en"selected="selected">

<input type="radio" name="show_variations" value="0"checked="checked">

<input type="text"id="form_step6_reference" name="form[step6][reference]" class="form-control"/>

<input type="text"id="form_step1_qty_0_shortcut" name="form[step1][qty_0_shortcut]" class="form-control"value="0"/>
```

**Same errors also found under:**
Admin->configure/shop/preferences/preferences/
Admin->/configure/shop/order-preferences/
Admin->/improve/international/taxes/
Admin->/improve/international/translations/settings/
Admin->/improve/shipping/preferences/
Admin->/improve/design/mail_theme/
Admin->/improve/design/cms-pages/
Admin->/sell/catalog/categories/
Admin->/sell/catalog/brands/
Admin->/sell/catalog/brands/addresses/

Tested with:
Firefox -> Dropdown's dosen't work and Add New Product will display all Taps in one page
Brave -> Completely unusable.

PHP 7.4.3
mysql  Ver 8.0.20-0
ubuntu0.20.04.1

**Comment by FrederikLive** [comment]
I solve the issue. It must be a memory allocation fault as I upgraded my Droplet, reinstalled and then everything worked perfectly.

**Comment by menno-maakt** [comment]
I've been experiencing this exact issue in 1.7.6.6, along with an inadvertent switching to maintenance mode. I updated to 1.7.6.8, issue went away.

But! Then it came back in 1.7.6.8! Also switched back to maintenance mode by itself. Weird. I did a full diff between the online version and a fresh download and no files had been changed. Twig templates were fine. Very strange.

I got it going again by switching to another PHP version, so, maybe opcache?
